src/main/resources/application.yml

```diff
@@ -205,3 +205,11 @@ aws:
     buckets:
       genotype:
         bucket: ${AWS_GENO_BUCKET}
+startup:
+  delay:
+    germplasm: ${germplasm_start_delay:2s}
```

Comment:
Can update these to caps as well when changed in env file

Reply:
updated to ALL CAPS